tests/functional/reload/24-reload-file-install.t

```diff
@@ -17,7 +17,7 @@
 #-------------------------------------------------------------------------------
 # Test reload triggers a fresh file install
 
-export REQUIRE_PLATFORM='loc:remote comms:?(tcp|ssh)'
```

Comment:
This would fail if run on a platform with a shared filesystem e.g. _remote_background_shared_ssh.